src/Button/native/Button.js

```diff
@@ -7,6 +7,9 @@ import Text from '../../Text/native';
 
 const styles = {
   buttonColor(props) {
+    if (props.kind === 'filled' && props.disabled) {
+      return `${props.theme.color[props.color]}aa`;
+    }
```

Comment:
you should be doing the opacity even if it's not a filled button

Reply:
wokay